cairo/tests/utils/args_gen.py

```diff
@@ -152,6 +152,10 @@ class Memory(bytearray):
     pass
 
 
+class MutableBloom(bytearray):
```

Comment:
why do we need to define the class?

Reply:
bytearray is not serialized by default as a list. we do like we did for memory.